builtin/providers/aws/resource_aws_emr_task_group.go

```diff
@@ -0,0 +1,114 @@
+package aws
+
+import (
+	"log"
+
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/emr"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsEMRTaskGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEMRTaskGroupCreate,
+		Read:   resourceAwsEMRTaskGroupRead,
+		Update: resourceAwsEMRTaskGroupUpdate,
+		Delete: resourceAwsEMRTaskGroupDelete,
+		Schema: map[string]*schema.Schema{
+			"cluster_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"instance_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
I see in the UPDATE method here that `instance_type` is not something the API supports updating; should this attribute then be marked as `ForceNew` so that we'll destroy->create on change?